test/server/cli-argument-parser-test.js

```
@@ -94,6 +94,27 @@ describe('CLI argument parser', function () {
         });
     });
 
+    describe('Speed', function () {
+        it('Should parse "--speed" option as a number ', function () {
+            return parse('--speed 0.01')
+                .then(function (parser) {
+                    expect(parser.opts.speed).eql(0.01);
+                });
+        });
+
+        it('Should raise an error if the "--speed" option value is not a number', function () {
+            return assertRaisesError('--speed yo', 'Speed should be a number between 0.01 and 1.');
+        });
+
+        it('Should raise an error if the "--speed" option value less than 0.01', function () {
+            return assertRaisesError('--speed 0', 'Speed should be a number between 0.01 and 1.');
+        });
+
+        it('Should raise an error if the "--speed" option value more than 0.01', function () {
```

`Should raise an error if the "--speed" option value more than 1`